subsys/usb/usb_transfer.c

```diff
@@ -13,7 +13,8 @@
 
 LOG_MODULE_REGISTER(usb_transfer, CONFIG_USB_DEVICE_LOG_LEVEL);
 
-#define MAX_NUM_TRANSFERS           4 /** Max number of parallel transfers */
+/** Max number of parallel transfers */
+#define MAX_NUM_TRANSFERS           CONFIG_USB_MAX_NUM_TRANSFERS
```

Comment:
I think this line could be removed and `CONFIG_USB_MAX_NUM_TRANSFERS` could be used directly without redefinition.

Reply:
Done.